tests/src/test/scala/tests.scala

```diff
@@ -27,7 +27,8 @@ class tests extends WordSpec with Matchers {
       import org.ensime.pcplod._
       withMrPlod("pcplodtest.scala") { pc =>
         pc.typeAtPoint('result) shouldBe Some("Option[Int]")
-        pc.typeAtPoint('test) shouldBe Some("(n: Int)freestyle.FreeS[F,Int]")
+        // Disabled test: the name of the parameter of `FreeS` is no longer known.
+        //pc.typeAtPoint('test) shouldBe Some("(n: Int)freestyle.FreeS[F,Int]")
```

Comment:
Not really. The point of the changes is that now, the placeholder variable `F[_]` that used to be part of the `@free`-annotated `trait` is now generated by the macro, as a fresh type name.